constants.py

```diff
@@ -7,3 +7,7 @@
 PROJECT_ROOT = Path(__file__).parent
 ASSETS_PATH = PROJECT_ROOT / 'tmp' / 'articles'
 CRAWLER_CONFIG_PATH = PROJECT_ROOT / 'scrapper_config.json'
+DOMAIN = "https://journals.kantiana.ru"
+HEADERS = {'user-agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 '
+                         '(KHTML, like Gecko) Chrome/99.0.4844.51 Safari/537.36'}
+PATH_RAW_END = "_raw.txt"
```

Comment:
why? I did not see any re-use of this constant

Reply:
I just thought it looked better, I deleted the constant.